R/benchTradePerf.R

```diff
@@ -0,0 +1,200 @@
+#' The \eqn{\bar{P}} is compared against a number of benchmark metrics, in order to assess its performance in terms of profit or loss.
+#' These methods are not strictly alternative, indeed each of them provides different insights and suffers limitations.
+#' 
+#' The performance is quantified by means of a \emph{Profit and Loss (PnL) metric}. 
+#' A positive PnL metric indicates that the trading strategy outperformed a chosen benchmark on average,
+#' viceversa negative values register an underperformance.
+#' 
+#' By and large, PnL metrics are computed as:
+#' 
+#' 
+#' where \eqn{P_{avg}} is the average execution price and \eqn{P_{B}} is a given benchmark price that can be a single next/present open/close price.
+#' It is worth stressing that they are expressed in basis points (bps) units.
+#' 
+#' A widely used one is the \emph{Volume Weighted Average Price (VWAP) benchamark}... 
+#' 
+#' A variation of the VWAP benchmark is given by the \emph{Participation Weighted Price (PWP) benchmark}.
+#' 
+#' 
+#' Lastly the \emph{Relative Performance Measure}, which differs from the PnL metrics above. It is expressed as:
+#' 
+#' 
+#' 
+#' @param Portfolio A portfolio name that points to a portfolio object structured with initPortf()
+#' @param Symbol A string identifying the traded symbol to benchmark
+#' @param side Numeric, either 1 or -1, that indicates the side of the trade. \code{side = 1} means "Buy" and \code{side = -1} is "Sell". Default is \code{side = 1}.
+#' @param benchmark A string or vector of strings providing the benchmark metric(s). One of 'MktBench', 'VWAP', 'PWP', 'RPM'. Default is #TODO.
+#' @param type A string specifying the type of the benchmark used. Relevant only for \code{benchmark = 'MktBench'} and \code{benchmark = 'VWAP'}. 
+#'             In the former case it only changes the output format, while in the latter it also specifies the VWAP to be used. If \code{benchmark = 'VWAP'}, default is \code{type = 'Txns'}
+#' @param MktData An xts object containing a 'MktPrice' and 'MktVolmn' required columns.
+#' @param POV A number between 0 and 1, specifying the POV rate.
+#' @param verbose A logical. It allows a RPM qualitative score appended. Default is \code{FALSE}.
+#'
+#' 
+#' @return 
+#' A list whose elements correspond in number to the length of the \code{benchmark} provided.
+#' Each element is a \code{data.frame} that can be one of the ones described below.
+#' 
+#' For \code{benchmark = 'MktBench'} it contains:
+#' \describe{
+#'      \item{\code{Symbol}: }{A string identifying the traded symbol to benchmark}
+#'      \item{\code{Side}: }{The \code{side} of the trades, as "Buy" or "Sell".}
+#'      \item{\code{Avg.Exec.Price}: }{Symbol transactions average execution price}
+#'      \item{\code{MktBench.*}: }{The benchmark and an arbitrary \code{type} provided as input (e.g. 'Open', 'Close').}
+#'      \item{\code{Performance}: }{The \emph{Benchmark PnL} performance, in bps.}
```

Comment:
No need for full stops when documenting parameters, unless of course you use 2 sentences
```suggestion
#'      \item{\code{Performance}: }{The \emph{Benchmark PnL} performance, in bps}
```

Reply:
Thank you for this documentation style suggestion